### ISSUE TITLE
How about solidjs, i'm curious about it

### ISSUE THREAD

**Issue by YUFENGWANG** [yyx990803/vue-svelte-size-analysis#1]
As it says, solidjs is both good at performance and size, which will win when compared to vue3?

https://www.solidjs.com/

**Comment by Rolanddoda** [comment]
Solidjs says is way faster than any framework.

![image](https://user-images.githubusercontent.com/18482346/124436259-126dd780-dd76-11eb-997b-e2ac7e41c3fb.png)

From the photo, it says that Svelte is way faster than Vue but this repo shows quite the opposite.

Strange how authors declare how fast is their Framework because it always seems correct but of course they have used different 
metrics. I really think that the metrics this repo uses are really valid for comparing frameworks.

**Comment by luukdv** [comment]
Exactly, it really depends on the metrics that are used. Also interesting: https://krausest.github.io/js-framework-benchmark/2021/table_chrome_91.0.4472.77.html
